robot/basestation/static/CSS/app.css

```diff
@@ -0,0 +1,143 @@
+html, body {
+    height: 100%;
+    background-color: #990000;
+}
+
+.container {
+    height: 100%;
+    min-height: 100%;
+}
+
+.fill {
+    width: 100%;
+    height: 100%;
+    min-height: 100%;
+    min-width: 100%;
+}
+
+.map {
+    height: 100vh;
+    min-height: 100vh;
+}
+
+.table_grp, .table_grp2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+.table_grp td, .table_grp2 td {
+    font-family: Arial, sans-serif;
+    font-size: 14px;
+    padding: 0;
+    border-style: none;
+    border-width: 0;
+    overflow: hidden;
+    word-break: normal;
+    border-color: #990000;
+}
+
+.table_grp th, .table_grp2 th {
+    font-family: Arial, sans-serif;
+    font-size: 14px;
+    font-weight: normal;
+    padding: 0;
+    border-style: solid;
+    border-width: 1px;
+    overflow: hidden;
+    word-break: normal;
+    border-color: #990000;
+    text-align: center;
+}
+
+.table_grp tr:last-child, .autoCtrLable {
+    border-bottom: 5px inset #05072F;
+    color: #6768A1;
+    font-weight: bold;
+    background-color: #101148;
+}
+
+.autoCtrLable {
+    padding: 12px 5px 12px 5px;
+    margin-top: 15px
+}
+
+.row, .row>* {
+    border: 0.5px solid #05072F;
```

Comment:
google CSS coding standards ([section 4.1.8](https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation)) dictate to use lowercase for hexadecimal notation, i.e.:
`#05072f` instead of `#O5072F`.